integration/pkg/api/oasis/options/options.go

```diff
@@ -0,0 +1,8 @@
+package options
```

Comment:
Is it really necessary to distinguish `genericoptions` and `options`  for `oasis`? I think they can be merged into `options` package.